config.json

```diff
@@ -1,8 +1,8 @@
 {
 	"serverRoot": "http://localhost:8000",
 	"port": 8000,
-	"dbtype": "sqlite3",
-	"dbconfig": "./focalboard.db",
+    "dbtype": "mysql",
```

Comment:
Why are you changing the defaults here to mysql?

Reply:
This was left from my testing. Pushing changes with that removed.

Reply:
Done.